cmd/wallet-sdk-gomobile/walleterror/error.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package walleterror define error model for go mobile api.
```

Comment:
`Package walleterror defines the error model for the Go mobile API.`

Reply:
done